tasks/concurrent.js

```diff
@@ -49,8 +49,11 @@ module.exports = function (grunt) {
 			});
 
 			if (opts.logConcurrentOutput) {
-				cp.stdout.pipe(padStream(' ', 4)).pipe(process.stdout);
-				cp.stderr.pipe(padStream(' ', 4)).pipe(process.stderr);
+				var delimiter = opts.delimiter || ' ';
```

Comment:
Making `delimiter` configurable is outside the scope of this PR and I'm not sure I even want that.